Author: Andreas Beckmann <anbe@debian.org>
Description: MDIO_EEE_1000T and MDIO_EEE_100TX are not defined in some older kernels

--- a/r8168_n.c
+++ b/r8168_n.c
@@ -24315,7 +24315,11 @@ rtl8168_init_software_variable(struct ne
         dev->max_mtu = tp->max_jumbo_frame_size;
 #endif //LINUX_VERSION_CODE >= KERNEL_VERSION(4,10,0)
         tp->eee_enabled = eee_enable;
+#ifdef MDIO_EEE_1000T
         tp->eee_adv_t = MDIO_EEE_1000T | MDIO_EEE_100TX;
+#else
+        tp->eee_adv_t = MDIO_AN_EEE_ADV_1000T | MDIO_AN_EEE_ADV_100TX;
+#endif
 }
 
 static void
